arm-resources/policy/2015-10-01-preview/swagger/policy.json

```diff
@@ -0,0 +1,693 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PolicyClient",
+    "version": "2015-10-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json",
+    "text/json"
+  ],
+  "produces": [
+    "application/json",
+    "text/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
```

Comment:
Just letting you know that Autorest does not support security definitions.Having them here doesnt hurt as long it is valid as per the swagger schema.